### DISCUSSION TITLE
"Download" as a tab on homescreen?

### DISCUSSION THREAD

**Discussion by woj-tek** [TeamNewPipe/NewPipe#11588] [Questions] [answered]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

Is it possible to add "Download" tab to home screen? It only lists "new", "plalist", "history" etc (similar to sidebar) but "Download" is missing from available options.

**Comment by ShareASmile** [comment]
You can find the relevant issue in issue tracker and follow it.

**Reply by woj-tek** [reply, marked as answer]
eh... found it! 

https://github.com/TeamNewPipe/NewPipe/issues/2565
